tensorflow/core/kernels/non_max_suppression_op.cc

```diff
@@ -778,5 +778,21 @@ REGISTER_KERNEL_BUILDER(
 
 REGISTER_KERNEL_BUILDER(Name("CombinedNonMaxSuppression").Device(DEVICE_CPU),
                         CombinedNonMaxSuppressionOp<CPUDevice>);
+// REGISTER FakeGPU op for CombinedNonMaxSuppression so that Funcdef'ed versions
+// of it can run.
+REGISTER_KERNEL_BUILDER(Name("CombinedNonMaxSuppression")
```

Comment:
Please split this part out to a separate PR so we can try to get it into 1.14. The rest of this PR will take longer time to review.

Reply:
Moved to #28808